plugins/permission-node/src/ServerPermissionClient.ts

```diff
@@ -76,22 +78,22 @@ export class ServerPermissionClient implements PermissionAuthorizer {
     this.tokenManager = options.tokenManager;
     this.permissionEnabled = options.permissionEnabled;
   }
+  async query(
+    queries: QueryPermissionRequest[],
+    options?: EvaluatorRequestOptions,
+  ): Promise<PolicyDecision[]> {
+    return (await this.isEnabled(options?.token))
```

Comment:
We already check the `permission.enabled` flag in the underlying `PermissionClient`, so there's an opportunity to remove that responsibility from this client and just use `this.isValidServerToken()` directly. However, this does make me realize the `permission.enabled` behavior is prone to breaking if a new `PermissionEvaluator` implementation is introduced. Something to keep an eye on or possible document on the interface definition

Reply:
Will revisit in a later change